Backend/admin/package.json

```diff
@@ -11,6 +11,7 @@
     "date-fns": "^2.11.1",
     "firebase": "^7.13.1",
     "lodash": "^4.17.15",
+    "moment": "^2.24.0",
```

Comment:
i think we have `date-fns` already no need for 2.
`date-fns` much faster then moment